web/data-crypto-sample/index.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+import { Des, Aes, TripleDes, pinBlock, pinBlockFormat3 } from "data-crypto";
+
+/*
+const {
+  Des,
+  Aes,
+  TripleDes,
+  pinBlock,
+  pinBlockFormat0,
+} = require("data-crypto");
+*/
+
+console.log('start')
+
+const pan = "1111222233334444";
+const pin = "1234";
+
+const block = pinBlockFormat3(pan, pin);
```

Comment:
As pointed in README.md, his implementation is questionable. I wonder if the author will get back to us though.